packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/migrating.doc.ts

```diff
@@ -19,6 +19,13 @@ const data: LandingTemplateSchema = {
       type: 'Generic',
       anchorLink: 'overview',
       title: 'Overview',
+      sectionNotice: [
```

Comment:
https://github.com/Shopify/cli/pull/4162 I was investigating migration and I remembered Stocky running into this issue where they were being prompted to "delete and make new" instead of migration. Then I recalled JS adding some kind of fix to it, so I searched the CLI pull requests with him as an author and found this one. I looked in the release list for the changelog with this PR, and found it.